custom_components/localtuya/light.py

```diff
@@ -366,10 +370,10 @@ def status_updated(self):
         self._state = self.dps(self._dp_id)
         supported = self.supported_features
         self._effect = None
-        if supported & SUPPORT_BRIGHTNESS:
+        if supported & SUPPORT_BRIGHTNESS and self.has_config(CONF_BRIGHTNESS):
```

Comment:
`SUPPORT_BRIGHTNESS` is set if config exists for brightness, so this change is redundant (see `supported_features` and you'll understand what I mean).

Reply:
This is not the only way, you can have RGB devices without white control, in that case support brightness is activated but via the setting in the color field.

Reply:
Oh, right, I missed that. Carry on then.